x-pack/plugins/apm/public/components/app/ServiceMap/cytoscapeOptions.ts

```diff
@@ -29,19 +29,20 @@ export const getSeverityColor = (nodeSeverity?: string) => {
   }
 };
 
-const getBorderColor: cytoscape.Css.MapperFunction<
-  cytoscape.NodeSingular,
-  string
-> = (el: cytoscape.NodeSingular) => {
-  const hasAnomalyDetectionJob = el.data('ml_job_id') !== undefined;
-  const nodeSeverity = el.data('anomaly_severity');
-  if (hasAnomalyDetectionJob) {
-    return getSeverityColor(nodeSeverity) || theme.euiColorMediumShade;
-  }
-  if (el.hasClass('primary') || el.selected()) {
-    return theme.euiColorPrimary;
-  }
-  return theme.euiColorMediumShade;
+const getBorderColorFn = (
+  theme: EuiTheme
```

Comment:
@balthazar: here `EuiTheme` is used as a type, so it is expected that to access styles you must `theme.eui.euiColorMediumShade`